README.md

```diff
@@ -224,6 +224,104 @@ messages with an unrecognised conversation ID on a dead letter queue.
+
+The endpoint also requires a JSON body that includes the needed patient NHS number. The format of the body should look like the following...
+
+   {
+      "resourceType": "Parameters",
+      "parameter": [
+         {
+            "name": "patientNHSNumber",
+            "valueIdentifier": {
+               "system": "https://fhir.nhs.uk/Id/nhs-number",
+               "value": "{{nhs-number-test-group-8}}"
+            }
+         },
+         {
+            "name": "includeFullRecord",
+            "part": [
+               {
+                  "name": "includeSensitiveInfomation",
+                  "valueBoolean": true
+               }
+            ]
+         }
+      ]
+   }
+
+Endpoint calling: 
+
+1. Initial request: If you successfully configure the endpoint described above and call it, you should receive a 202-accepted response. This means the adaptor has received the request and is making the relevant requests.
+2. Polling the request: after receiving a 202 response, we recommend polling the endpoint at regular intervals using an increasing call gap strategy. Each poll can return the following responses:-
+   1. 204 No content: this response indicates that we are still processing the requests / waiting for the EHR message response.
+   2. 200 Success: this response indicates we have successfully received and converted the EHR to JSON; you will also receive the FHIR bundle in the response's body.
+   3. 400,404,500,501: The endpoint can return all these possible error codes. These will all provide a detailed error with an operationOutcome JSON model response in the body. This looks like...
+ 
+   {
+      "resourceType": "OperationOutcome",
+      "meta": {
+         "profile": [
+            "https://fhir.nhs.uk/STU3/StructureDefinition/GPConnect-OperationOutcome-1"
+         ]
+      },
+      "issue": [
+         {
+            "severity": "error",
+            "code": "exception",
+            "details": {
+               "coding": [
+                  {
+                     "system": "https://fhir.nhs.uk/STU3/ValueSet/Spine-ErrorOrWarningCode-1",
+                     "code": "PATIENT_NOT_FOUND",
+                     "display": "GP2GP - Patient is not registered at the practice"
+                  }
+               ]
+            }
+         }
+      ]
+   }
+
+### /$gpc.ack
+
+The ack endpoint is the final endpoint to call once you are happy that the EHR record you have received is acceptable.
+If you do not call this endpoint after receiving an EHR from the migratestructuredrecord enpoint, then you risk the losing practise triggering off the manual postal transfer.
+
+The following is required to call this endpoint:-
+
+- CONVERSATION_ID: The id associated with the patient transfer request.
+- CONFIRMATION_RESPONSE: you can provide the following status:-
+   - ACCEPTED: This will tell the sending incumbent that you are happy with the received EHR.
```

Comment:
AG said the following for PS_MSG_4 "Just to highlight that only ACCEPTED and FAILED_TO_INTEGRATE are the only two responses should only ever be returned by the /$gpc.ack endpoint as NME’s will be automatically integrating the patient record." Should we be documenting the other responses?

Reply:
I was tempted to leave them since we do support them in code, however, we could mis this later on so I've removed them in preparation to our change